src/transformers/modeling_longformer.py

```diff
@@ -823,13 +839,13 @@ def _init_weights(self, module):
 )
 class LongformerModel(LongformerPreTrainedModel):
     """
-    This class overrides :class:`~transformers.RobertaModel` to provide the ability to process
-    long sequences following the selfattention approach described in `Longformer: the Long-Document Transformer
-    <https://arxiv.org/abs/2004.05150>`__ by Iz Beltagy, Matthew E. Peters, and Arman Cohan. Longformer selfattention
+    This class copied code from :class:`~transformers.RobertaModel` and overwrote standard self-attention with longformer self-attention to provide the ability to process
```

Comment:
(nittiest of all nits)

present tense verbs for docstrings.
copies, overwrites. The class is a thing that exists in the present, at least from the reader's perspective.